doc/api/dns.md

```diff
@@ -147,6 +139,12 @@ on some operating systems (e.g FreeBSD 10.1).
 <!-- YAML
 added: v0.11.14
 -->
+- `address` {string}
+- `port` {number}
+- `callback(err, hostname, service)` {Function}
```

Comment:
I don't know if it's customary to include the arguments in this line. Rather if I recall correctly it's just <code>&#96;callback&#96; {Function}</code>.